src/public/js/Togethernet/systemMessage.js

```diff
@@ -1,12 +1,57 @@
-export const addSystemMessage = (systemMsg) => {
-  $('#systemMessage').find('span').text(systemMsg);
-  $('#systemMessage').show();
-  $(document).one('mouseup', clearSystemMessage);
-  $(document).one('keydown', clearSystemMessage);
+export const addSystemConfirmMessage = ({
+  msgHeader = '',
+  msgBody = '',
+  msgFooter = '',
+  yayText = '',
+  nayText = '',
+  yayBtn = 'Continue',
+  yayBtnTitle = 'Continue',
+  nayBtn = 'Stop',
+  nayBtnTitle = 'Stop',
+  nayLink = "parent.open('https://togethernet.app')",
+}) => {
+  $('#systemConfirmMessage').find('h1').text(msgHeader);
+  $('#systemConfirmMessage').find('p').text(msgBody);
+  $('#systemConfirmMessage').find('h6').text(msgFooter);
+  $('#systemConfirmMessage').find('.yay-container p').text(yayText);
+  $('#systemConfirmMessage').find('.nay-container p').text(nayText);
+  $('#systemConfirmMessage').find('button.yay').text(yayBtn);
+  $('#systemConfirmMessage').find('button.yay').attr('title', yayBtnTitle);
+  $('#systemConfirmMessage').find('button.nay').text(nayBtn);
+  $('#systemConfirmMessage').find('button.nay').attr('title', nayBtnTitle);
+  $('#systemConfirmMessage').find('button.nay').attr('onClick', nayLink);
```

Comment:
I wouldn't pass in the attribute for onClick - into the systemMessage function. For security + accessibility + readability purposes. 
as we discussed, you could do either
1. have both a button and a link in the system message modal, the button being always hidden. If no href is passed in, hide the link. If you want to use the button, add the event handling & unhide the button where you want to use it
2. have a button styled like a link. Add the event handlingwhere you want to use it

Reply:
resolved.